dev-support/hbase_nightly_yetus.sh

```diff
@@ -66,7 +66,7 @@ YETUS_ARGS=("--branch=${BRANCH_NAME}" "${YETUS_ARGS[@]}")
 YETUS_ARGS=("--tests-filter=${TESTS_FILTER}" "${YETUS_ARGS[@]}")
 YETUS_ARGS=("--ignore-unknown-options=true" "${YETUS_ARGS[@]}")
 # Why are these not being picked up from hbase-personality?
-YETUS_ARGS=("--proclimit=10000" "${YETUS_ARGS[@]}")
+YETUS_ARGS=("--proclimit=12500" "${YETUS_ARGS[@]}")
```

Comment:
If the above change for `PROC_LIMIT` works, this line can be dropped.

Reply:
Yeah. For now they needed. Let me file an issue.